### PR TITLE
1828 implment designs for email template

### LINKED ISSUE #1828: Implement designs for email template
### Overview

Design team has created an email template design and now we need to implement it for our Contact Page emails.

### Action Items

- [x] In the 311-Data repo, create a new directory: `311-data/templates`
- [x] Create a new file: `311-data/templates/contact-us.html`
- [x] Implement the referenced design
  - [x] contain the work within `contact-us.html`, do not use external style sheets (see R1)
- [x] Send a test email with using the Send HTML In Gmail plugin (see R2)
- [x] Upload a screenshot into the `Screenshot after proposed changes` section of this post

### Resources/Instructions

- R1: How to send html emails in Gmail [blog post](https://htmlemail.io/blog/how-to-send-html-emails-in-gmail)
- R2: Chrome Plugin for embedding HTML in emails: [link to chrome web store](https://chromewebstore.google.com/detail/send-html-in-gmail/ipaogmomajeofkdpcmmjgilnblolomhb)

##### Figma Info

- [Figma design file link](https://www.figma.com/design/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=13452-6294&t=EanmSPB9TZU10CIM-1)
- Related design ticket: https://github.com/hackforla/311-data/issues/1763

<details><summary>Screenshot before proposed changes</summary>
<p>

n/a

</p>
</details>

<details><summary>Screenshot after proposed changes</summary>
<p>
image

</p>
</details>

### PR DESCRIPTION
**Pull request by Brandoncyu**
Fixes #1828 

  - [x] Up to date with `main` branch
  - [x] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: templates/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+    <header style="background-color: #29404F; text-align: center; ">
+        <img src="../assets/311Logo.png" width="75" style="padding: 10px">
```
[the remark this answers is not in the document]
Comment (Brandoncyu) [reply templates/contact-us.html]: Done

File: templates/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+        <p>
+            Sincerely,
+            <br>
+            311 Data Volunteers at Hack for LA.
+        </p>
+    </div>
+    <footer style="padding-top: 50px; text-align: center">
+        <p>...</p>
+        <img src="../assets/hack_for_la_logo.png"  width="75">
```
[the remark this answers is not in the document]
Comment (Brandoncyu) [reply templates/contact-us.html]: Done

**Comment by Brandoncyu** [comment]
Made adjustments. New template works:
image

**Comment by ryanfchase** [comment]
I am seeing a weird black line in your image. I'm not seeing anything in the code that would warrant that. I've tested your code and the html seems to be rendering as expected. Leaving my screenshot so that we can have design lead review.

image

**Comment by Brandoncyu** [comment]
The black line I believe is the cursor where the keyboard is. This is because this is in draft status.

**Comment by Brandoncyu** [comment]
I've tried it again and I don't see it. Perhaps it happened when I copied pasted the photo:

image

**Comment by Brandoncyu** [comment]
I've tried it again and I can't seem to repro the black line in the image. Perhaps it was a copy paste issue for the photo?

image

**Comment by ryanfchase** [comment]
> I've tried it again and I can't seem to repro the black line in the image. Perhaps it was a copy paste issue for the photo?

Sounds good! Thanks for confirming. This ticket will still await design lead review. Marking it in our [agenda ](https://github.com/hackforla/311-data/issues/1545#issuecomment-2372711386)for tonight.

**Review by ryanfchase — APPROVED** [review]
This is looking good, approved 👍

**Review by ryanfchase — APPROVED** [review]
Font family was changed to Arial. Confirmed the font is sans-serif. Feedback has been implemented, this is approved and ready to merge.

**Pull request merged**